webapp/settings.py

```diff
@@ -17,33 +20,51 @@
 # (see: http://stackoverflow.com/a/15383766/613540)
 SECRET_KEY = 'SECRET_KEY_INSECURE_PLACEHOLDER'  # !! CHANGE ME !!
 
-# See https://docs.djangoproject.com/en/dev/ref/contrib/
-INSTALLED_APPS = ['django_versioned_static_url']
-
 ALLOWED_HOSTS = ['*']
+DEBUG = False
 
-MIDDLEWARE_CLASSES = []
-
-ROOT_URLCONF = 'webapp.urls'
-WSGI_APPLICATION = 'webapp.wsgi.application'
 LANGUAGE_CODE = 'en-us'
 TIME_ZONE = 'UTC'
 USE_I18N = False
 USE_L10N = False
 USE_TZ = False
-STATIC_ROOT = 'static'
+
+WSGI_APPLICATION = 'webapp.wsgi.application'
+ROOT_URLCONF = 'webapp.urls'
+APPEND_SLASH = False
+REMOVE_SLASH = True
+STATIC_ROOT = "static"
 STATIC_URL = '/static/'
-STATICFILES_FINDERS = ['django_static_root_finder.StaticRootFinder']
-TEMPLATE_DIRS = ['templates']
 
-# See http://tinyurl.com/django-context-processors
-TEMPLATE_CONTEXT_PROCESSORS = [
-    "django.core.context_processors.static",  # {{ STATIC_URL }}
-    "django_asset_server_url.asset_server_url",  # {{ ASSET_SERVER_URL }}
+WHITENOISE_MAX_AGE = 31557600
```

Comment:
I'm not sure if this should be in here. It sets cache headers to tell browsers to cache static files for a year, which is pretty dangerous if we think those files might change before then. It only really works if we definitely are using something like django_versioned_static_url for linking to all static resources that ever change. I know we [do have it installed](https://github.com/WillMoggridge/django-blueprint/blob/1fd0b581aaa89cae8476ecad1f5c226eba1b126d/requirements/standard.txt#L3) and we [are using it in the one link we have in this project](https://github.com/WillMoggridge/django-blueprint/blob/1fd0b581aaa89cae8476ecad1f5c226eba1b126d/templates/index.html#L8) but is that enough to trust that any forks of this project will continue to honour that? Maybe it is...